### PR TITLE
feat: parsing error endpoint

### PR DESCRIPTION
**Pull request by diivi**
### What
- Add an endpoint to get all parsing errors related to a project
- Modify project list endpoint to include error_count for every project.

### Fixes bug(s)
- #156

### CODE REVIEW

File: backend/editor/entries.py [hunk 1]
```diff
@@ -244,10 +244,13 @@ def close_project(self):
...
         """
         query = """
             MATCH (n:PROJECT)
-            WHERE n.status = "OPEN" RETURN n
+            OPTIONAL MATCH (error_node:ERRORS {{branch_name: n.branch_name, id: n.id}})
+            WHERE n.status = "OPEN"
+            RETURN n, size(error_node.errors) AS error_count
             ORDER BY n.created_at
         """
         result = get_current_transaction().run(query)
```
Comment (alexgarel) [inline backend/editor/entries.py:256]: Here we should transform the result to include our error count in result:

```suggestion
        result = get_current_transaction().run(query)
        # pack errors count in result
        result = [dict(node, errors_count=num_errors) for node, num_errors in result]

```

Comment (diivi) [reply backend/editor/entries.py:256]: Ah, great! I was trying to do this in the query itself 😅

**Review by alexgarel — APPROVED** [review]
Great @diivi !

**Comment by alexgarel** [comment]
@diivi merged with main, as there where conflicting changes from @aadarsh-ram 

I also saw my suggestion was not correct (in fact previously list where already nested, so I kept the nesting…)

**Comment by alexgarel** [comment]
First merged request @diivi, kudos :tada: !

**Pull request merged**